packages/theme/src/generated/themes-fs/atb-theme/dark.css

```diff
@@ -0,0 +1,755 @@
+  --color-palette-brand-50-foreground-secondary: #555e65;
+  --color-palette-brand-50-foreground-disabled: #a9aeb1;
+  --color-palette-brand-100-background: #eed2db;
+  --color-palette-brand-100-foreground-primary: #000000;
+  --color-palette-brand-100-foreground-secondary: #555e65;
+  --color-palette-brand-100-foreground-disabled: #a9aeb1;
+  --color-palette-brand-200-background: #e4b8c6;
+  --color-palette-brand-200-foreground-primary: #000000;
+  --color-palette-brand-200-foreground-secondary: #555e65;
+  --color-palette-brand-200-foreground-disabled: #a9aeb1;
+  --color-palette-brand-300-background: #d691a7;
+  --color-palette-brand-300-foreground-primary: #000000;
+  --color-palette-brand-300-foreground-secondary: #555e65;
+  --color-palette-brand-300-foreground-disabled: #a9aeb1;
+  --color-palette-brand-400-background: #c76b89;
+  --color-palette-brand-400-foreground-primary: #000000;
+  --color-palette-brand-400-foreground-secondary: #555e65;
+  --color-palette-brand-400-foreground-disabled: #a9aeb1;
+  --color-palette-brand-500-background: #b74166;
+  --color-palette-brand-500-foreground-primary: #000000;
+  --color-palette-brand-500-foreground-secondary: #555e65;
+  --color-palette-brand-500-foreground-disabled: #a9aeb1;
+  --color-palette-brand-600-background: #a51140;
+  --color-palette-brand-600-foreground-primary: #ffffff;
+  --color-palette-brand-600-foreground-secondary: #e1e7eb;
+  --color-palette-brand-600-foreground-disabled: #a9aeb1;
+  --color-palette-brand-700-background: #7d0d31;
+  --color-palette-brand-700-foreground-primary: #ffffff;
+  --color-palette-brand-700-foreground-secondary: #e1e7eb;
+  --color-palette-brand-700-foreground-disabled: #a9aeb1;
+  --color-palette-brand-800-background: #5c0a24;
+  --color-palette-brand-800-foreground-primary: #ffffff;
+  --color-palette-brand-800-foreground-secondary: #e1e7eb;
+  --color-palette-brand-800-foreground-disabled: #a9aeb1;
+  --color-palette-brand-900-background: #380616;
+  --color-palette-brand-900-foreground-primary: #ffffff;
+  --color-palette-brand-900-foreground-secondary: #e1e7eb;
+  --color-palette-brand-900-foreground-disabled: #a9aeb1;
+  --color-palette-brand-950-background: #380616;
+  --color-palette-brand-950-foreground-primary: #ffffff;
+  --color-palette-brand-950-foreground-secondary: #e1e7eb;
+  --color-palette-brand-950-foreground-disabled: #a9aeb1;
+  --border-radius-small: 4;
+  --border-radius-regular: 8;
+  --border-radius-circle: 20;
+  --border-width-slim: 1;
+  --border-width-medium: 2;
+  --typography-ios-font: 'SF Pro Text';
+  --typography-ios-number: -0.3100000023841858;
+  --typography-android-font: Roboto;
+  --typography-android-number: -0.5;
+  --typography-web-font: Roboto;
+  --typography-web-number: -0.5;
+  --spacing-x-small: 4;
+  --spacing-small: 8;
+  --spacing-medium: 12;
+  --spacing-large: 20;
+  --spacing-x-large: 24;
+  --icon-size-x-small: 12;
+  --icon-size-small: 16;
+  --icon-size-normal: 20;
+  --icon-size-large: 26;
```

Comment:
Have you considered how we will add units to these values? Today we have e.g. `--border-radius-small: 0.25rem;`

Reply:
Great catch! The type of these variables was not configured correctly in Style Dictionary. Fixed that now and added the `size/pxToRem` transformer in Style Dictionary. Should be correct now